Automattic/portkey-android
# Update naming to stories

Use stories consistently throughout the codebase, and also rename the base app (at the moment effectively a demo app) to Loop.

Counterpart WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/12546

To test:
* Make sure everything runs
* I made some changes to the tooling store to reflecting the new naming, so confirm that CI is passing and an APK is generated
